### PR TITLE
add recorded trades time from first trade made in !trades command

### PR DESCRIPTION
**Pull request by idinium96**
[chat1](https://user-images.githubusercontent.com/47635037/76429371-e9d80680-63e9-11ea-8030-4ad61c58cc49.png)
[moment.js: Time from now](https://momentjs.com/docs/#/displaying/fromnow/)

### CODE REVIEW

**Comment by idinium96** [comment]
> Instead of calling it operation time, just say that it has traded recorded from x days ago

Alright, I'll change that to "All trades made were recorded from".

**Comment by idinium96** [comment]
[chat2](https://user-images.githubusercontent.com/47635037/76461582-207a4500-641b-11ea-8958-21e8dc5b8b57.PNG)

File: src/classes/Commands.ts [hunk 1]
```diff
@@ -1228,15 +1228,30 @@ export = class Commands {
...
-        const offerData = this.bot.manager.pollData.offerData;
+        const timeSincePoll = this.bot.manager.pollData.timestamps;
+        const timeSince = timeSincePoll[Object.keys(timeSincePoll)[0]];
+
+        let operateMessage = '';
+        const operateSince = moment(timeSince * 1000).fromNow();
+        operateMessage =
+            operateSince === 'Invalid date'
+                ? '\n Just started. Waiting for the first trade.'
+                : '\n All trades made were recorded from ' + operateSince;
```
Comment (Nicklason) [inline src/classes/Commands.ts]: I think the line is too big. Change it to "Trades recorded from x days ago". Make sure to use the pluralize module. The code should be something like this:

`"Trades recorded from " + pluralize(totalDays, true) + " ago"`

Comment (idinium96) [reply src/classes/Commands.ts]: Now my codes were broken. Also, I'm surprised that you want me to pluralize a number. The moment().fromNow() already pluralizing itself. That "days" count only appear if moment().fromNow() output changed from (a few second ago/x seconds ago/a minute ago/ x minutes ago/ an hour ago/ x hours ago/a day ago/ x days ago) to (a month ago/ x months ago/ a year ago/ x years ago).

It's ok Nick. I'll close this if it's not necessary.

Comment (Nicklason) [reply src/classes/Commands.ts]: Yes I know what it does that, but that does not tell you exactly how many days ago the bot got trades from.

**Comment by idinium96** [comment]
It will show the exact days.
[chat3-final](https://user-images.githubusercontent.com/47635037/76579303-a1623b00-6506-11ea-9701-f28906eadc9a.png)

**Comment by idinium96** [comment]
This will do?
[another final](https://user-images.githubusercontent.com/47635037/76580023-5695f280-6509-11ea-96e6-a360c1e124da.PNG)

**Comment by idinium96** [comment]
[as you wish](https://user-images.githubusercontent.com/47635037/76580981-bb9f1780-650c-11ea-94e7-08d097172d4f.PNG)

**Comment by idinium96** [comment]
Do I need to close this an make another pull request with the latest commit?

File: src/classes/Commands.ts [hunk 2]
```diff
@@ -1228,15 +1228,28 @@ export = class Commands {
...
+
+        let reply = '';
+        reply += !timeSince ? '\n Just started. Waiting for the first trade.' : '\n Trades recorded from ';
 
+        const totalDaysInSeconds = now.valueOf() - timeSince;
+        let totalDays = Math.round((totalDaysInSeconds / 86400) * 1);
+        if (totalDays < 1) {
+            totalDays = 0;
+        }
+        reply += totalDays + ' ' + pluralize('day', totalDays) + ' ago';
```
Comment (Nicklason) [inline src/classes/Commands.ts]: Pluralize can add the days infront of it by giving it true ad the third argument.

Comment (idinium96) [reply src/classes/Commands.ts]: so, `pluralize('day', totalDays, true)` ?

**Comment by idinium96** [comment]
Yeah.. this look nicer with true on the third argument of the pluralize function.
looks nicer

**Comment by Nicklason** [comment]
> Yeah.. this looks nicer with true on the third argument of the pluralize function.
> looks nicer

You need to remove where you add the number yourself.

Also, why did you close the issue?

**Comment by idinium96** [comment]
moved to #382
Well... I've done too many commits that aren't necessary and might be just increasing my commit point. better open the new one with the good one.

**Pull request closed without merging**